lang/java/reef-checkpoint/src/main/java/org/apache/reef/io/checkpoint/fs/FSCheckpointServiceConfiguration.java

```diff
@@ -39,12 +39,12 @@
 import java.io.IOException;
 
 /**
- * ConfigurationModule for the FSCheckPointService.
+ * ConfigurationModule for the FSCheckpointService.
  * This can be used to create Evaluator-side configurations of the checkpointing service.
  */
 @DriverSide
 @Public
-public class FSCheckPointServiceConfiguration extends ConfigurationModuleBuilder {
+public class FSCheckpointServiceConfiguration extends ConfigurationModuleBuilder {
```

Comment:
This is a `public` and `@Public` API. We cannot change it without first marking it as `@Deprecated`